app/serializers/oembed_serializer.rb

```diff
@@ -57,10 +59,10 @@ def html
   end
 
   def width
-    instance_options[:width]
+    (instance_options[:width] || DEFAULT_WIDTH).to_i
   end
 
   def height
-    instance_options[:height]
+    Array.new(instance_options[:height].to_i).detect(&:positive?)
```

Comment:
Wouldn't the following be clearer and more straightforward?
```suggestion
    instance_options[:height].presence&.to_i
```